src/main/kotlin/com/chattriggers/ctjs/minecraft/wrappers/Scoreboard.kt

```diff
@@ -124,10 +112,6 @@ object Scoreboard : CTWrapper<MCScoreboard?> {
         scoreboardNames = scoreboard.getAllPlayerScores(sidebarObjective).map(::Score).toMutableList()
     }
 
-    fun resetCache() {
-        needsUpdate = true
-    }
```

Comment:
Why get rid of this? We shouldn't do the work of calculating the scoreboard values if it isn't needed

Reply:
Fair enough, but can we then force the user to call `updateLines()` themself, so that the `if` constructs can be left out?

Reply:
Not sure why we would want to do that. It seems like the only change here should be uncommenting the call to `resetCache`

Reply:
should be good now